test/sslapitest.c

```diff
@@ -98,12 +98,14 @@ static char *dhfile = NULL;
 static int is_fips = 0;
 static int fips_ems_check = 0;
 
-#define LOG_BUFFER_SIZE 2048
+#if !defined(OPENSSL_NO_SSLKEYLOG_CB)
```

Comment:
This might happen to work in CI, but I don't think it is correct. The variables are also used in blocks guarded by the three other conditions I listed.

Reply:
:). I turned the. cut and paste (dang, even the comment!) into a function ...